lib/src/html_generator.dart

```diff
@@ -523,6 +528,13 @@ String renderMarkdown(String markdown, {nestedContext}) {
   return doc.body.innerHtml;
 }
 
+/// Convert the given plain text into HTML.
+String renderPlainText(String text) {
+  if (text == null) return '';
+
+  return "<code class='fixed'>${text.trim()}</code>";
```

Comment:
`pre` has some selectors that make it look like this:

![screen shot 2015-05-17 at 11 20 10 am](https://cloud.githubusercontent.com/assets/1269969/7671479/d1486b16-fc86-11e4-82fe-af85b40ae2b5.png)

w/ `code.fixed`, it looks like this:

![screen shot 2015-05-17 at 11 19 19 am](https://cloud.githubusercontent.com/assets/1269969/7671480/e133e85c-fc86-11e4-8483-182d46292f76.png)

This is really just a fallback for (pretty infrequent) readme.txt files (and no readme.md).